### ISSUE TITLE
Sync only when system is idle

### ISSUE THREAD

**Issue by sathishvj** [ethereum/mist#1223]
Feature Request:

If I was a non power user, I need to use the ethereum-wallet only once in a while.  However, keeping ethereum-wallet/mist/geth running all the time keeps even my high-end computer quite busy and loaded.  Would it be possible to have a sync-on-idle feature that runs the sync process only when the computer is reasonably idle?

**Comment by luclu** [comment]
@sathishvj although this is a very good idea, the upcoming **lightclient**-capability ([LES-protocol](https://github.com/zsfelfoldi/go-ethereum/wiki/Light-Ethereum-Subprotocol-(LES))) will unburden the client from the current workload (Initial sync time down to under 2 min) and thus improve the situation for most users that would enjoy this feature.

On the other hand their might be some configurations where an sync-on-idle feature would still be desirable although the implementation would require significant additions to the current stack:
 - as syncing is done by the ethereum-client (node) every implementation would require to incorporate it
 - the web3.js API would require the appropriate commands
 - and finally Mist would need some GUI

In the light of the upcoming LES functionality and the current saturation of developer-resources on long lists of critical integrations and improvements this is on low priority for now.

Though the best way to get this starting might be to create an [EIP](https://github.com/ethereum/EIPs).
